code/modules/antagonists/nukeop/nukeop.dm

```diff
@@ -521,12 +521,113 @@
-	return common_part + disk_report + challenge_report
+
+	var/post_report
+
+	var/war_declared = FALSE
+	for(var/obj/item/circuitboard/computer/syndicate_shuttle/board as anything in GLOB.syndicate_shuttle_boards)
+		if(board.challenge)
+			war_declared = TRUE
+
+	var/force_war_button = ""
+
+	if(war_declared)
+		post_report += "<b>War declared.</b>"
+		force_war_button = "\[Force war\]"
+	else
+		post_report += "<b>War not declared.</b>"
+		var/obj/item/nuclear_challenge/war_button = war_button_ref?.resolve()
+		if(war_button)
+			force_war_button = "<a href='?_src_=holder;[HrefToken()];force_war=[REF(war_button)]'>\[Force war\]</a>"
+		else
+			force_war_button = "\[Cannot declare war, challenge button missing!\]"
+
+	post_report += "\n[force_war_button]"
+	post_report += "\n<a href='?_src_=holder;[HrefToken()];give_reinforcement=[REF(src)]'>\[Send Reinforcement\]</a>"
+
+	var/final_report = ..()
+	final_report += disk_report
+	final_report += post_report
+	return final_report
+
+#define SPAWN_AT_BASE "Nuke base"
+#define SPAWN_AT_INFILTRATOR "Infiltrator"
+
+/datum/team/nuclear/proc/admin_spawn_reinforcement(mob/admin)
+	if(!check_rights_for(admin.client, R_ADMIN))
+		return
+
+	var/infil_or_nukebase = tgui_alert(
+		admin,
+		"Spawn them at the nuke base, or in the Infiltrator?",
+		"Where to reinforce?",
+		list(SPAWN_AT_BASE, SPAWN_AT_INFILTRATOR, "Cancel"),
+	)
+
+	if(!infil_or_nukebase || infil_or_nukebase == "Cancel")
+		return
+
+	var/tc_to_spawn = tgui_input_number(admin, "How much TC to spawn with?", "TC", 0, 100)
+
+	var/list/nuke_candidates = poll_ghost_candidates(
+		"Do you want to play emergency synciate reinforcements?",
+		ROLE_OPERATIVE,
+		ROLE_OPERATIVE,
+		30 SECONDS,
+		POLL_IGNORE_SYNDICATE,
+	)
+
+	nuke_candidates -= admin // may be easy to fat-finger say yes. so just don't
+
+	if(!length(nuke_candidates))
+		tgui_alert(admin, "No candidates found.", "Recruitment Shortage", list("OK"))
+		return
+
+
+	var/turf/spawn_loc
+	if(infil_or_nukebase == SPAWN_AT_INFILTRATOR)
+		// I'd love to use areas_by_type but it's a unique area
+		var/area/spawn_in = (locate(/area/shuttle/syndicate/hallway) in GLOB.areas) || (locate(/area/shuttle/syndicate) in GLOB.areas)
+		var/list/turf/options = list()
+		for(var/turf/open/open_turf in spawn_in?.get_contained_turfs())
+			if(open_turf.is_blocked_turf())
+				continue
+			if(istype(open_turf, /turf/open/floor/fakespace)) // goofy nuke diorama
+				continue
+			options += open_turf
+
+		if(length(options))
+			spawn_loc = pick(options)
+		else
+			infil_or_nukebase = SPAWN_AT_BASE
+
+	if(infil_or_nukebase == SPAWN_AT_BASE)
+		spawn_loc = pick(GLOB.nukeop_start)
+
+	var/mob/dead/observer/picked = pick(nuke_candidates)
+	var/mob/living/carbon/human/nukie = new(spawn_loc)
+	picked.client.prefs.safe_transfer_prefs_to(nukie, is_antag = TRUE)
+	nukie.key = picked.key
+
+	var/datum/antagonist/nukeop/antag_datum = new()
```

Comment:
```suggestion
	var/datum/antagonist/nukeop/antag_datum = new
```

Reply:
actually does this stuff remain in nullspace or does `add_antag_datum` add it to the mind's content or something

Reply:
wat? It's a datum

Reply:
it's not un-necessary, we aren't feeding any args so we don't need the parenthesis. it's totally unlike a spacing/formatting nit. it's a pattern that @Mothblocks wanted to be put in anyways, i could care less

Reply:
also on the whole nullspace thing i was confused- we hold a ref to the mob on the `/datum` so that's whatever. for some reason i thought it would autocleanup